src/components/data_comps_mct/datm/src/datm_comp_mod.F90

```diff
@@ -366,6 +366,7 @@ subroutine datm_comp_init(Eclock, x2a, a2x, &
        kptem = mct_aVect_indexRA(a2x,'Sa_ptem')
        kshum = mct_aVect_indexRA(a2x,'Sa_shum')
        kdens = mct_aVect_indexRA(a2x,'Sa_dens')
+       kuovern = mct_aVect_indexRA(a2x,'Sa_uovern')
```

Comment:
Do you actually need this? It doesn't look like you refer to kuovern anywhere, so it doesn't need to be retrieved (and it may not exist outside of e3sm).

Reply:
Thanks @billsacks . Yes, I don't need it. I have updated  it in my recent commit.